### PR TITLE
WELD-1940 Backport Probe development tool

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

**Comment by weld-tester** [comment]
Triggering build using a merge of e971841179e5d33262850b72653d32c6119a9ce6 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 791 outcome was FAILURE using a merge of e971841179e5d33262850b72653d32c6119a9ce6 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/791
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/791

**Comment by jharting** [comment]
We'll need to suppress probe integration tests on WildFly until WF has proper Weld 2.3 support (I am not going to switch to custom WF builds for PR testing in the meantime).

**Comment by mkouba** [comment]
@jharting Yep, that makes sense. I will temporary set the tomcat profile as a default.

**Comment by weld-tester** [comment]
Triggering build using a merge of 58b6bbea9fef43693ce586f34baf7aac29668bcd on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 793 is now running using a merge of 58b6bbea9fef43693ce586f34baf7aac29668bcd on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/793

**Comment by weld-tester** [comment]
Build 793 outcome was SUCCESS using a merge of 58b6bbea9fef43693ce586f34baf7aac29668bcd on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/793
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/793

**Comment by weld-tester** [comment]
Build 793 outcome was SUCCESS using a merge of 58b6bbea9fef43693ce586f34baf7aac29668bcd on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/793
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/793

**Comment by weld-tester** [comment]
Build 793 outcome was SUCCESS using a merge of 58b6bbea9fef43693ce586f34baf7aac29668bcd on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/793
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/793

**Comment by weld-tester** [comment]
Build 793 outcome was SUCCESS using a merge of 58b6bbea9fef43693ce586f34baf7aac29668bcd on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/793
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/793

File: probe/core/pom.xml [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.weld</groupId>
+            <artifactId>weld-core-impl</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jboss.spec.javax.servlet</groupId>
+            <artifactId>jboss-servlet-api_3.0_spec</artifactId>
```
Comment (jharting) [inline probe/core/pom.xml]: The 2.3 branch has been updated recently to use `jboss-servlet-api_3.1_spec` and other more up to date API artifacts.  See https://github.com/weld/core/commit/690e77ba016433acfe77df9356435f92618c45df Update and rebase this PR accordingly.

**Comment by weld-tester** [comment]
Triggering build using a merge of 8214d1961272f30b342c235196f6aae12087f054 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 799 is now running using a merge of 8214d1961272f30b342c235196f6aae12087f054 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/799

**Comment by weld-tester** [comment]
Build 799 outcome was SUCCESS using a merge of 8214d1961272f30b342c235196f6aae12087f054 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/799
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/799

**Comment by weld-tester** [comment]
Build 799 outcome was SUCCESS using a merge of 8214d1961272f30b342c235196f6aae12087f054 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/799
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/799

**Comment by weld-tester** [comment]
Build 799 outcome was SUCCESS using a merge of 8214d1961272f30b342c235196f6aae12087f054 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/799
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/799

**Comment by jharting** [comment]
retest this please

**Comment by weld-tester** [comment]
Triggering build using a merge of 8214d1961272f30b342c235196f6aae12087f054 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 800 is now running using a merge of 8214d1961272f30b342c235196f6aae12087f054 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/800

**Comment by weld-tester** [comment]
Build 800 outcome was SUCCESS using a merge of 8214d1961272f30b342c235196f6aae12087f054 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/800
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/800

**Comment by weld-tester** [comment]
Triggering build using a merge of 4a0ba52560c75ed729b1b44404569c3d0ce938a1 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 801 is now running using a merge of 4a0ba52560c75ed729b1b44404569c3d0ce938a1 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/801

**Comment by weld-tester** [comment]
Build 801 outcome was SUCCESS using a merge of 4a0ba52560c75ed729b1b44404569c3d0ce938a1 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/801
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/801

**Comment by weld-tester** [comment]
Build 801 outcome was SUCCESS using a merge of 4a0ba52560c75ed729b1b44404569c3d0ce938a1 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/801
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/801

**Comment by weld-tester** [comment]
Build 801 outcome was SUCCESS using a merge of 4a0ba52560c75ed729b1b44404569c3d0ce938a1 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/801
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/801

**Comment by weld-tester** [comment]
Build 801 outcome was SUCCESS using a merge of 4a0ba52560c75ed729b1b44404569c3d0ce938a1 on branch 2.3:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/801
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/801

**Comment by jharting** [comment]
Merged, thanks!

**Pull request merged**